src/app/content/studyGuides/hooks/loadMore.spec.ts

```diff
@@ -173,16 +197,37 @@ describe('loadMore', () => {
 
     expect(loadingSpy).toHaveBeenCalled();
     expect(highlightClient).toHaveBeenCalledTimes(3);
-    expect(dispatch).lastCalledWith(receiveSummaryStudyGuides(response2, {pagination: null, filters}));
+    expect(dispatch).lastCalledWith(receiveSummaryStudyGuides(response2, {pagination: null}));
```

Comment:
@TomWoodward This is the other failing test. When I log the actions dispatched this is the last one but it never happens according to the test.

Reply:
when you run this test it doesn't think `receiveSummaryStudyGuides` has been called here? i don't see how that could be possible other than throwing an error or hitting the noop you added for the open flag. have you tried running just this one test to make sure there is no strange output? what does the state look like?